forte/trainer/base/base_trainer_new.py

```diff
@@ -0,0 +1,258 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
```

Comment:
The final name won't be `base_trainer_new` right

Reply:
Yes. And we cannot just rename it to `base_trainer` as this file is already existing and some other modules depends on this old base trainer. Do you have any suggestions about this?

Reply:
Well it is up to you then.